src/pymodaq_plugins_keithley/daq_viewer_plugins/plugins_0D/daq_0Dviewer_Keithley2100.py

```diff
@@ -31,19 +26,12 @@ class DAQ_0DViewer_Keithley2100(DAQ_Viewer_base):
     :type params: dictionary list
     """
 
-    # Read configuration file
-    for instr in config["Keithley", "2100"].keys():
-        if "INSTRUMENT" in instr:
-            resources_list += [config["Keithley", "2100", instr, "rsrc_name"]]
-    logger.info("resources list = {}".format(resources_list))
-
     params = comon_parameters + [
         {
             "title": "Resources",
             "name": "resources",
-            "type": "list",
-            "limits": resources_list,
-            "value": resources_list[0],
+            "type": "str",
+            "value": rsrc_name, 
```

Comment:
where is the rsrc_name defined? I only say a type declaration line 11?

Reply:
Deleted rsrc_name and fixed value.